website/content/partials/datasource/amazon/ami/DatasourceOutput-not-required.mdx

```diff
@@ -0,0 +1,13 @@
+<!-- Code generated from the comments of the DatasourceOutput struct in datasource/amazon/ami/data.go; DO NOT EDIT MANUALLY -->
+
+- `id` (string) - The ID of the AMI.
+
+- `name` (string) - The name of the AMI.
+
+- `creation_date` (string) - The date of creation of the AMI.
+
+- `owner` (string) - The AWS account ID of the owner.
+
+- `owner_name` (string) - The owner alias.
+
+- `tags` (map[string]string) - The key/value combination of the tags assigned to the AMI.
```

Comment:
Oh really cool ! I wonder if we should do something about that 'not-required' part in the name

Reply:
It would be nice, I'll have a look at this but for another PR.